### DISCUSSION TITLE
Lighthouse Documentation

### DISCUSSION THREAD

**Discussion by CRhodes-123** [bitcraze/discussions#319] [Q&A]
Hello everyone, 
I am a university student working with crazyflies for research, and I am trying to set up two lighthouse base stations currently, but because they are the V2 version I have been having some troubles getting the calibration and geometry data from them. I remember that there was documentation on how to get this data manually at this link, but it now gives a 404 for me: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/2021.01/functional-areas/lighthouse/setting_up/

I was wondering if the instructions on how to get the calibration or geometry data manually could be found anywhere else, or if that page could be restored because it would help me out tremendously.

Thanks,
Collin Rhodes

**Comment by krichardsson** [comment]
We compile documentation from our repositories and render it for our web but recently we decided to not build the full history, this is why the page you are looking for does not exist any more. The good thing is that the source of the page (in mark down, so easy to read) is in the crazyflie-firmware repo, [you can find it here](https://github.com/bitcraze/crazyflie-firmware/blob/2021.01/docs/functional-areas/lighthouse/setting_up.md) 

Otherwise the documentation that is available can be found here:
[Getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/)
[Tech documentation](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/lighthouse/)

The instructions for getting the calibration data seems to have been removed at some point when we have re-written the documentation. Do you think it would be valuable to add the instructions again?

**Reply by CRhodes-123** [reply]
I think it would be helpful to add the instructions, although the naming conventions of setting up vs. getting started can be a little confusing. I think making one set of instructions for setting up the hardware/lighthouse base stations and one for setting up software/the crazyflie controller or something along those lines may be a little easier to follow.

Thank you for directing me to the crazyflie-firmware repo, I did not know about this and it is very helpful!

Best,
Collin R.
